scripts/tools/zap/zap_bootstrap.sh

```diff
@@ -1,64 +0,0 @@
-#!/bin/bash
```

Comment:
What is the replacement for this for local development?

Reply:
Or said it another way: bootstraping zap if zap is installed locally is a zap development workflow rather than a matter one.